### PR TITLE
Migrate DDR Component ARK's to corresponding RDR FileSets; RDR-337.

### PR DESCRIPTION
**Pull request by coblej**

### CODE REVIEW

**Comment by coblej** [comment]
I decided not to write tests for this script.  It's essentially a one-time use script (run once for each migrated collection).  It provides a 'dryrun' option to report on the actions that the script would take when run for real.  I imagine this option will be used prior to the actual run for each collection.  Once the migration is complete and verified, I expect that this script will be removed from the code base.

File: lib/tasks/rdr.rake [hunk 1]
```diff
@@ -9,4 +9,15 @@ namespace :rdr do
       system("git", "push", "origin", tag)
     end
   end
+
+  namespace :migration do
+    desc 'Migrate DDR Component ARKs to RDR FileSets (EXPORT_DIR, DRYRUN, LOGGER)'
```
[the remark this answers is not in the document]
Comment (coblej) [reply lib/tasks/rdr.rake]: @dchandekstark Good point.  Revised PR coming up.

[the remark this answers is not in the document]
Comment (ad96) [reply lib/tasks/rdr.rake]: In our work yesterday for JIRA-340, we have a Logger attr in the script but not in the rake task.

[the remark this answers is not in the document]
Comment (coblej) [reply lib/tasks/rdr.rake]: @dchandekstark @ad96 Fixed

**Comment by coblej** [comment]
Closing this PR because it fails to account for a particular scenario we need to accommodate.  Will resubmit once I've figured out how to do that.

**Pull request closed without merging**